tools/scoring/score/dimensions/entity_identification.py

```diff
@@ -27,6 +27,9 @@
 class EntityIdentification(Dimension):
   """Quantifies whether the correct entities
   were included in the proposed file."""
+
+  category = DimensionCategories.COMPLEX
```

Comment:
same here how do you know it is Complex?
do you need to add extra explanation here? 
https://github.com/google/digitalbuildings/blob/master/tools/scoring/score/constants.py#L26

Reply:
Commented above; it is complex because the score is derived from information which is effectively not processed beforehand. More context is provided in `parse_config.py` and in `dimension.py`, and in fact the comment in `constants.py` is not strictly accurate (as a multi-map is not built for Entity Connection Identification). I will work on expanding the explanation and unifying it between files.

Reply:
thanks !

Reply:
are you going to do it in this PR? or have a todo ?

Reply:
I'll do it here.